VkApp/VkApp/Posts/VkPostsViewController.swift

```diff
@@ -20,32 +27,36 @@ final class VkPostsViewController: UIViewController {
         print("EXIT")
     }
 }
-
     // MARK: - UITableViewDataSource
 extension VkPostsViewController: UITableViewDataSource {
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        3
+        cellTypes.count
```

Comment:
```suggestion
        4
```
4 ячейки всегда по заданию